hash.py

```diff
@@ -18,7 +18,7 @@
 import sys
 
 
-FILENAME_PREFIX = 'tf-aws-lambda-'
+FILENAME_PREFIX = 'terraform-aws-lambda-'
```

Comment:
Changing the name here means that old zip files with the old prefix (generated by the old version of this module) may stick around in the `.terraform` directory. I don't think it's worth writing the code to clean them up.